src/executables/sysrepo-plugind.c

```diff
@@ -196,7 +196,7 @@ daemon_init(int debug, sr_log_level_t log_level)
     }
 
     /* redirect standard files to /dev/null */
-    fd = open("/dev/null", O_RDWR, 0);
+    fd = open("/dev/null", O_RDWR | O_NOFOLLOW, 0);
```

Comment:
I really hope `/dev/null` is not some malicious symlink :) However, are you sure it cannot be a symbolic link on some regular distribution?

Reply:
Good point.  No need for the restriction here.